content/embeds/tryout-redisearch-2.md

```diff
@@ -0,0 +1,93 @@
+## Creating indexes
+
+Let's create a new index called "myIdx". When you define the index, you
+must pass in the structure of the data you are adding to the index.
+In this example, we have four things we input: title, body, url, and
+value. In this example, we have three TEXT and one NUMERIC values. The
+title has a weight of 5.0.
+
+1. Connect to redis.
+
+    ```sh
+    $ redis-cli -p 12543
+    127.0.0.1:12543>
+    ```
+
+1. Create the schema with the command:
+
+    ```sh
+    127.0.0.1:12543> FT.CREATE myIdx PREFIX 1 "doc:" SCORE_FIELD "value" SCHEMA title TEXT body TEXT url TEXT value NUMERIC
+    ```
+
+    This command indexes the hashes that have the prefix "doc".
+    By default, all hashes get a score of 1.0 but here the index uses the `value` field as the score for this hash.
+
+{{< note >}}
+For databases in a Redis Cloud Essentials subscription, you need to add the index name to the document key as a tag to make sure that the index and the documents are located on the same shard:
```

Comment:
You need to add the index name as a field in the document. Is that correct? The language "add the index name to the document key as a tag" is unclear.

Also, break into two sentences.

Example:

For Redis Cloud Essentials databases, you need to add the index name as a field in the document. This will ensure that a document and its index will reside on the same shard.

Reply:
This entire section can go away, once we have RediSearch 2.0 in Redis Cloud essentials @MeirShpilraien please confirm

Reply:
@K-Jo in cloud essential its RediSearch light with a single shard always right? and if someone wants to move to RCP he will do it with replica of? if this is the case then yes the tag is not needed.